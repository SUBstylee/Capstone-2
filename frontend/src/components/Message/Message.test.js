import { render } from '@testing-library/react';
import Message from './Message';

test("renders Message component without crashing", () => {
    const { getByText } = render(
        <Message variant='alert' children='test message' />
    );

    expect(getByText("test message")).toBeInTheDocument();
});

test("matches snapshot", function () {
    const { asFragment } = render(
        <Message />
    );
    expect(asFragment()).toMatchSnapshot();
});