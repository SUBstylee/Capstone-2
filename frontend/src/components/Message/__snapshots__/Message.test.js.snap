// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="fade alert alert-info show"
    role="alert"
  />
</DocumentFragment>
`;
